templates/form/mobilepay.liquid

```diff
@@ -1,10 +1,16 @@
 <div class="container">
   {% include 'form/error' %}
   <form action="{{ model.redirect_url }}" method="POST" target="_top" class="autosubmit">
-    <input type="hidden" name="SessionToken" value="{{ model.session_token }}" />
+    <input type="hidden" name="{{ model.token_input_name }}" value="{{ model.session_token }}" />
```

Comment:
Value variable hasn't changed and hence it should have no impact on current nor previous releases. Use of addresses feature will require updating any custom branding with above change.

(Tested locally with current branding without issues)